GET_STARTED.md

```diff
@@ -0,0 +1,442 @@
+
+#### Activate member
+
+```bash
+curl "https://ccf_service_url/gov/ack/update_state_digest" -X POST --cacert service_cert.pem --key member0_privk.pem --cert member0_cert.pem --silent | jq > request.json
+cat request.json
+/opt/ccf/bin/scurl.sh "https://ccf_service_url/gov/ack"  --cacert service_cert.pem --signing-key member0_privk.pem --signing-cert member0_cert.pem --header "Content-Type: application/json" --data-binary @request.json
+```
+
+#### New member proposal
+
+```json
+{
+  "actions": [
+    {
+      "name": "set_member",
+      "args": {
+        "cert": "member_cert",
+        "encryption_pub_key": "member_encryption_pub_key"
+      }
+    }
+  ]
+}
+```
+
+### Network Governance: Adding users
+
+Users directly interact with the application running in CCF. Their public identities should be voted in by members before they are allowed to issue requests
+
+Once a CCF network is successfully started and an acceptable number of nodes have joined, members should vote to open the network to Users.
+First, the identities of trusted users should be generated,see [Generating Member Keys and Certificates](https://microsoft.github.io/CCF/main/governance/adding_member.html#generating-member-keys-and-certificates) and [Adding Users docs](https://microsoft.github.io/CCF/main/governance/open_network.html)
+
+#### New user proposal
+
+```json
+{
+  "actions": [
+    {
+      "name": "set_user",
+      "args": {
+        "cert": "user_cert"
+      }
+    }
+  ]
+}
+```
+
+### Network Governance: Application deployment
+
+The native format for JavaScript applications in CCF is a [JavaScript application bundle](https://microsoft.github.io/CCF/main/build_apps/js_app_bundle.html), or short app bundle. A bundle can be wrapped directly into a governance proposal for deployment.
+
+#### Application deployment proposal
+
+```json
+{
+  "actions": [
+    {
+      "name": "set_js_app",
+      "args": {
+        "bundle": {
+          "metadata": { "endpoints": {} },
+          "modules": []
+        }
+      }
+    }
+  ]
+}
+```
+
+### Network Governance: Open network for users
+
+Once users are added to the opening network, members should create a proposal to open the network,
+Other members are then able to vote for the proposal using the returned proposal id.
+
+Once the proposal has received enough votes under the rules of the Constitution (ie. ballots which evaluate to true), the network is opened to users.
+It is only then that users are able to execute transactions on the deployed application.
+
+#### Open network proposal
+
+```json
+{
+  "actions": [
+    {
+      "name": "transition_service_to_open",
+      "args": {
+        "next_service_identity": "service_cert"
+      }
+    }
+  ]
+}
+```
+
+---
+
+## Development Dependencies Installation
+
+- CCF Setup
+- NodeJS
+- NPM
+
+### CCF Install
+
+To install the ccf runtime please check [Install CCF](https://microsoft.github.io/CCF/main/build_apps/install_bin.html)
+
```

Comment:
At this point I would just point them at the other repo rather than copying the steps as this will get out of date quickly